src/iuse.cpp

```diff
@@ -7357,8 +7357,8 @@ int iuse::weather_tool( player *p, item *it, bool, const tripoint & )
         }
         const oter_id &cur_om_ter = overmap_buffer.ter( p->global_omt_location() );
         /* windpower defined in internal velocity units (=.01 mph) */
-        int windpower = int( 100.0f * get_local_windpower( weatherPoint.windpower + vehwindspeed,
-                             cur_om_ter, g->is_sheltered( g->u.pos() ) ) );
+        double windpower = int( 100.0f * get_local_windpower( weatherPoint.windpower + vehwindspeed,
+                                cur_om_ter, g->u.pos(), g->is_sheltered( g->u.pos() ) ) );
```

Comment:
Function accepts `player *p` as parameter. Use that instead of `g->u`:
`cur_om_ter, p->pos(), g->is_sheltered( p->pos() ) ) );`